app/models/manageiq/providers/amazon/cloud_manager/event_catcher/stream.rb

```diff
@@ -159,12 +160,29 @@ def queue_url_to_arn(queue_url)
   # @param [Aws::SQS::Types::Message] message
   def parse_event(message)
     event = JSON.parse(JSON.parse(message.body)['Message'])
-    $log.info("#{log_header} Found SNS Message with message type #{event["messageType"]}")
-    return unless event["messageType"] == "ConfigurationItemChangeNotification"
+
+    if event["messageType"] == "ConfigurationItemChangeNotification"
+      # Aws Config Events
+      event["eventType"]    = parse_event_type(event)
+      event["event_source"] = :config
+    elsif event.fetch_path("detail", "eventType") == "AwsApiCall"
+      # CloudWatch with CloudTrail for API requests Events
+      event["eventType"]    = event.fetch_path("detail", "eventName")
+      event["event_source"] = :cloud_watch_api
+    elsif event["detail-type"] == "EC2 Instance State-change Notification"
```

Comment:
so only event_source is what I am adding, to know how the parse the event in parser. Rest is AWS API.